app/app.go

```diff
@@ -300,6 +306,12 @@ func New(
 	app.StakingKeeper = *stakingKeeper.SetHooks(
 		stakingtypes.NewMultiStakingHooks(app.DistrKeeper.Hooks(), app.SlashingKeeper.Hooks()),
 	)
+	epochsKeeper := epochskeeper.NewKeeper(appCodec, keys[epochstypes.StoreKey])
```

Comment:
Sir, are we missing anything here?

@nguyenvuong1122000 

PS:  Nice work.

Reply:
Oh no, sir. That variable is use to call setHooks func. And now we don't use any Hook.

Reply:
i use it here :  [[code](https://github.com/notional-labs/dig/blob/91295406d2523bdd0dc7c79d977fc23937574cd9/app/app.go#L310-L313)](https://github.com/notional-labs/dig/blob/91295406d2523bdd0dc7c79d977fc23937574cd9/app/app.go#L310-L313)